lib/services/local/users.go

```diff
@@ -903,10 +912,23 @@ func (s *IdentityService) GetWebauthnSessionData(ctx context.Context, user, sess
 		return nil, trace.BadParameter("missing parameter sessionID")
 	}
 
+	key := sessionDataKey(user, sessionID)
 	item, err := s.Get(ctx, sessionDataKey(user, sessionID))
-	if err != nil {
+	if trace.IsNotFound(err) {
+		return nil, trace.NotFound("item %q is not found", string(key))
```

Comment:
Could go even more opaque imo:

```suggestion
		return nil, trace.NotFound("webauthn session data not found")
```